features/step_definitions/index_scope_steps.rb

```diff
@@ -36,7 +36,11 @@
   expect(page).to     have_css ".scopes .scope-group-#{group} .#{name2}"
 end
 
-Then"I should see an empty group with the scope {string}" do |name|
+Then "I should see an empty group with the scope {string}" do |name|
```

Comment:
Sorry, I forgot to mention this. I landed this separately, so if you could rebase on top of latest master to remove this bit from the final diff, that'd be great :+1:.

Reply:
I will do it.